roles/bootnode-parity/tasks/main.yml

```diff
@@ -1,21 +1,5 @@
 ---
-- name: Create directory parity_data/keys/NetworkName
```

Comment:
We don't need to create a folder with the keys anymore?

Reply:
Not for `bootnode` role. Parity automatically creates this folder and puts `address_book.json` there. However, for `validator` we need to create it and place validator keys before starting the node.